src/sync/installation.ts

```diff
@@ -157,7 +157,7 @@ const updateJobStatus = async (
 			logger.warn(err, "Error sending the `complete` event to JIRA");
 		}
 	} else {
-		logger.info("Sync status for installationId=%d is pending", installationId);
+		logger.info({ installationId }, "Sync status for installationId=%d is pending");
```

Comment:
This will be logged as "Sync status for installationId=**%d** is pending" , installationId was a string format parameter